test/e2e/framework/create.go

```diff
@@ -362,18 +362,18 @@ func (f *Framework) patchItemRecursively(item interface{}) error {
 	case *apps.StatefulSet:
 		f.PatchNamespace(&item.ObjectMeta.Namespace)
 		if err := f.PatchContainerImages(item.Spec.Template.Spec.Containers); err != nil {
-			return errors.Wrapf(err, "%T", f)
```

Comment:
Why?

It's generally a good idea to explain the "why" in the commit message or in the code, because then reviewers don't need to ask ;-}

Hmm, it looks like this commit fixes the previous one. I think you can just squash the two.